projects/cardiovascular-patient-app/src/assets/env.template.js

```diff
@@ -1,4 +1,10 @@
-window.env = {
-    clientId: "${CLIENT_ID}",
-  };
-  
\ No newline at end of file
+// window.env = {
+//     clientId: "${CLIENT_ID}",
+//     appTitle: "${APP_TITLE}"
+//   };
+
+(function (window) {
+  window["env"] = window["env"] || {};
+  window["env"]['clientId'] = "${CLIENT_ID}";
+  window["env"]['appTitle'] = "${APP_TITLE}";
+})(this);
```

Comment:
please clean up env.template.js so it matches the local env.js

Reply:
Okay @semicolin